### DISCUSSION TITLE
Where to put Z Min plug with this Firmware

### DISCUSSION THREAD

**Discussion by mechano** [mechano/Kingroon_KP3S_Pro_Marlin#1] [Announcements]
Remember how to connect the Z end sto cable of the BL Touch. 
For this firmware unplug the actual Z Min switch, leave unplugged.
And connect the 2 wires plug of the BL Touch to the Z Max free green socket.

![image](https://github.com/mechano/Kingroon_KP3S_Pro_Marlin_Firmware/assets/2874499/80cacb74-eec4-4e59-b74e-27e442853719)

**Comment by stdakov** [comment]
Hi  I have installed the marlin2 but I can not find how to setup the correct Z offset. Can you tell me how to do it from the UI

**Comment by stdakov** [comment]
And can you enable Z offset wizard in the firmware

**Comment by mechano** [comment]
Search into menus it's in two positions, one is Motion->Bed leveling and
another one is Configuration->Probe Z-offset
I think you have to set, store settings, switch off and on again. I don't
remember wich one permit on the fly correction.

Il giorno mar 24 ott 2023 alle ore 16:39 Stanislav Dakov <
***@***.***> ha scritto:

> Hi I have installed the marlin2 but I can not find how to setup the
> correct Z offset. Can you tell me how to do it from the UI
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mechano/Kingroon_KP3S_Pro_Marlin_Firmware/discussions/1#discussioncomment-7371071>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAV5ZA345MHSSOHGXBFPFBLYA7HIZAVCNFSM6AAAAAA5XVMWMCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGNZRGA3TC>
> .
> You are receiving this because you authored the thread.Message ID:
> <mechano/Kingroon_KP3S_Pro_Marlin_Firmware/repo-discussions/1/comments/7371071
> @github.com>
>


-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***

**Reply by stdakov** [reply]
can you compile again the firmware with these settings enabled 
![image](https://github.com/mechano/Kingroon_KP3S_Pro_Marlin_Firmware/assets/13567110/7b600452-bdb4-4f11-940c-e10b0e96477c)

**Reply by stdakov** [reply]
This is in Configuration_adv.h file

**Comment by mechano** [comment]
Yes check in few minutes I'm compiling it...
I'll add to description this feature when ready.


Il giorno mar 24 ott 2023 alle ore 16:47 Stanislav Dakov
***@***.***> ha scritto:
>
> This is in Configuration_adv.h file
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>



-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***

**Comment by mechano** [comment]
Ok I've tested and works perfectly.

I create the archive soon you can download.

Il giorno mar 24 ott 2023 alle ore 16:51 Roberto Giuffre`
***@***.***> ha scritto:
>
> Yes check in few minutes I'm compiling it...
> I'll add to description this feature when ready.
>
>
> Il giorno mar 24 ott 2023 alle ore 16:47 Stanislav Dakov
> ***@***.***> ha scritto:
> >
> > This is in Configuration_adv.h file
> >
> > —
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> > You are receiving this because you authored the thread.Message ID: ***@***.***>
>
>
>
> --
> Roberto Giuffrè
> tel. +39-328-9140378
> email: ***@***.***
> pec: ***@***.***



-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***

**Reply by stdakov** [reply]
I have tested it too works great! Thank you so much! My offset is -1.75 :)

**Comment by mechano** [comment]
Strange I have -0.75 but I use PEI double side with buildtak


Il giorno mar 24 ott 2023 alle ore 17:21 Stanislav Dakov <
***@***.***> ha scritto:

> I have tested it too works great! Thank you so much! My offset is -1.75 :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mechano/Kingroon_KP3S_Pro_Marlin_Firmware/discussions/1#discussioncomment-7371515>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAV5ZA4RBF6CUA7BE3DVMBTYA7MGXAVCNFSM6AAAAAA5XVMWMCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGNZRGUYTK>
> .
> You are receiving this because you authored the thread.Message ID:
> <mechano/Kingroon_KP3S_Pro_Marlin_Firmware/repo-discussions/1/comments/7371515
> @github.com>
>


-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***
